src/components/Table/TableRow/TableRow.scss

```diff
@@ -0,0 +1,21 @@
+.moonstone-table_row {
+    border-bottom: 1px solid var(--color-gray40);
+
+    &:hover {
```

Comment:
We have to prevent the hover for the row in the table header.

Reply:
I'm thinking about the most simple way to organize the `TableHead` and `TableRow` components with regard to this. I was thinking about having the `TableHead` component include a `tr` since it should always be used with a `tr` element anyway. But then, `TableHead` and `TableBody` would differ in how they're used, and, `TableHead` would differ a little from the way the HTML implementation looks as well - I'm not sure if this is actually an issue, though.